### PR TITLE
EASY fixes

### PR DESCRIPTION
**Pull request by daviandr**
I think I got it

### CODE REVIEW

File: src/main/java/edu/oregonstate/cs361/battleship/Easy.java [hunk 1]
```diff
@@ -8,7 +8,7 @@
  */
 public class Easy extends BattleshipModel{
     //place ships
-    private String[] shipName = {"aircraftCarrier", "battleship", "submarine", "dhingy", "clipper", "fisher"};
+    private String[] shipName = {"submarine", "dhingy", "clipper", "fisher"};
```
Comment (nickywongdong) [inline src/main/java/edu/oregonstate/cs361/battleship/Easy.java]: Hey i think you accidentally removed some ships, this destroy our game logic, you should definitely consider adding it back.

File: src/main/java/edu/oregonstate/cs361/battleship/Main.java [hunk 2]
```diff
@@ -12,6 +12,8 @@
 
 public class Main {
 
+    public BattleshipModel bm;
```
Comment (nickywongdong) [inline src/main/java/edu/oregonstate/cs361/battleship/Main.java]: You shouldn't be creating instance of battleship here, this is handled in main

**Comment by jakevolkman** [comment]
Oooooh weeee [https://youtu.be/-PPB14T3eSs](url)

File: src/main/java/edu/oregonstate/cs361/battleship/Easy.java (hunk 1 above)
Comment (daviandr) [reply src/main/java/edu/oregonstate/cs361/battleship/Easy.java]: No, we don't need those ships.

Comment (daviandr) [reply src/main/java/edu/oregonstate/cs361/battleship/Easy.java]: jk I got it.

**Pull request closed without merging**